Cogs/AdminCommands.py

```diff
@@ -28,6 +40,19 @@ async def downloadcorgis(self, ctx, amount):
     @commands.command(help='`-clear AMOUNT` to clear AMOUNT messages\n`-clear all` to clear all messages from this channel')
     @commands.has_permissions(administrator=True)
     async def clear(self, ctx, amount=''):
+        """Clears a specific number of messages from a guild
+        Take in user input for the number of messages they would like to get cleared. If the amount is 'all',
+        clear a very large number of messages from the server. If the amount is blank, tell user how to more
+        properly use the command. Otherwise, send message confirming how many messages are being cleared and log it.
+        Purge the appropriate number of messages from the channel.
+
+        Args:
+            amount (int): Number of messages to be removed
```

Comment:
`amount` should be type `str`, not `int`

Reply:
Resolved by 9c40e9a